src/arrayMethodSort.js

```diff
@@ -4,8 +4,35 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const defaultSort = (a, b) => {
+    if (a.toString() > b.toString()) {
+      return 1;
+    } else if (a.toString() < b.toString()) {
```

Comment:
Try to avoid using `if else` condition, you can simplify it into the 2 `if` conditions.